fusestore/store/src/meta_service/sled_vartype_tree_test.rs

```diff
@@ -966,8 +966,8 @@ async fn test_as_multi_types() -> anyhow::Result<()> {
     let tc = new_sled_test_context();
     let db = &tc.db;
     let tree = SledVarTypeTree::open(db, tc.config.tree_name("foo"), true).await?;
-    let aslog = tree.as_type::<sledkv::Logs>();
-    let asmeta = tree.as_type::<StateMachineMeta>();
+    let log_tree = tree.key_space::<sledkv::Logs>();
+    let sm_meta = tree.key_space::<StateMachineMeta>();
```

Comment:
I prefer `sm_meta_tree` as of above is named `log_tree`.

Reply:
Right the names in unit tests aren't well thought.

I prefer `sm_meta` than `sm_meta_tree` since it is shorter...And the concept `tree` is not that important in this context.

I named it `log_tree` but not `logs` because there is a conflict...

What say you bro?

Reply:
All right, you convinced me.

Reply:
> All right, you convinced me.

😭 Thanks for your understanding... It's not easy for me to find a perfect name...